h/search/query.py

```diff
@@ -193,6 +193,20 @@ def __call__(self, search, params):
         )
 
 
+class SharedAnnotationsFilter:
```

Comment:
I'm always caught by surprise about how this API works, with these callable classes to represent different filters.

:shrug:  Anyway this fits in the overall current design.